CONTRIBUTING.md

```diff
@@ -51,39 +51,18 @@ fully covered are very welcome!
 
 ### Supporting both Python 2 and 3
 
-Most of the functionality in Dipy works on both Python 3 and Python 2. Please
+Most of the functionality in DIPY works on both Python 3 and Python 2. Please
 follow the instructions [here](http://dipy.org/devel/python3.html) to
 write code that works on both versions.
 
 ### Code style
 
-Code contributions should be formatted according to the
-[PEP8](https://www.python.org/dev/peps/pep-0008/) style guide. There are
-[software systems](https://pypi.python.org/pypi/pep8) that will check your code
-for PEP8 compliance, and most text editors can be configured to check the
-compliance of your code with PEP8.
+Code contributions should be formatted according to the [DIPY Coding Style Guideline](./doc/devel/coding_style_guideline.rst).
```

Comment:
Eventually, once the style guide gets to the website, we should link to the website docs, instead?